app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,4 @@
+<footer>
+    <p class="pull-right" style="padding-right: 10px;"><a href="#">Back to top</a></p>
+    <p class="pull-left" style="padding-left: 10px;">© 2016 Derek Stride, Justin Krol</p>
```

Comment:
I think you'll need `pull-xs-left` and `pull-xs-right` from http://v4-alpha.getbootstrap.com/components/utilities/#responsive-floats it looks like `pull-right` and `pull-left` are mixins.